### PR TITLE
Add a Updating the TLS certificate section

### PR DESCRIPTION
**Pull request by gotsysdba**

### CODE REVIEW

**Review by markxnelson — APPROVED** [review]
please get review from @rhondaday too

**Comment by gotsysdba** [comment]
@rhondaday, can you give this change a once-over and approve if good?

File: docs-source/mbaas/content/setup/_index.md
Comment (rhondaday) [inline docs-source/mbaas/content/setup/_index.md:90]: Change "Self-Signed" to self-signed (no quote; no caps)

File: docs-source/mbaas/content/setup/_index.md
Comment (rhondaday) [inline docs-source/mbaas/content/setup/_index.md:94]: Change Address to address

**Review by rhondaday — CHANGES_REQUESTED** [review]

**Review by markxnelson — DISMISSED** [review]
LGTM

File: docs-source/mbaas/content/setup/_index.md
Comment (rhondaday) [inline docs-source/mbaas/content/setup/_index.md:90]: In the first sentence, change "TLS" to "Transport Layer Security (TLS)".

It looks like there may be an extra space between sentences. There should only be one space between sentences.

File: docs-source/mbaas/content/setup/_index.md
Comment (rhondaday) [inline docs-source/mbaas/content/setup/_index.md:94]: Change to:

1. Ensure that your Domain Name System (DNS) entry ...

**Review by rhondaday — COMMENTED** [review]
Just a couple of comments.

**Review by rhondaday — APPROVED** [review]
Looks good.

**Pull request merged**
